zeppelin-web/src/app/credential/credential.controller.js

```diff
@@ -21,15 +21,28 @@ angular.module('zeppelinWebApp').controller('CredentialCtrl', function($scope, $
   $scope.credentialEntity = '';
   $scope.credentialUsername = '';
   $scope.credentialPassword = '';
-  
+
   $scope.updateCredentials = function() {
+    if (_.isEmpty($scope.credentialEntity.trim()) ||
+        _.isEmpty($scope.credentialUsername.trim()) ||
+        _.isEmpty($scope.credentialPassword.trim())) {
+      BootstrapDialog.alert({
+        closable: true,
+        message: 'The values must be filled.'
```

Comment:
How about "Username\Entity can not be empty"?

Also, this will not allow user to have `" "` whitespace as a password, right? For username that make sense, but I think we should not have such restriction on the password